Mathlib/LinearAlgebra/FiniteDimensional/Basic.lean

```diff
@@ -393,6 +393,22 @@ theorem ofInjectiveEndo_left_inv (f : V →ₗ[K] V) (h_inj : Injective f) :
     ((ofInjectiveEndo f h_inj).symm : V →ₗ[K] V) * f = 1 :=
   LinearMap.ext <| (ofInjectiveEndo f h_inj).symm_apply_apply
 
+variable {V' : Type*} [AddCommGroup V'] [Module K V'] [FiniteDimensional K V']
+omit [FiniteDimensional K V]
+
+/-- An injective linear map between finite-dimensional space of equal rank
```

Comment:
Doesn't the `omit` above mean that only one space is finite dimensional? Either way I think there is a typo here, grammatically something is missing.